### PR TITLE
Slave message formatting

### PR DESCRIPTION
**Pull request by seandst**
I started with the slave message formatting, and threw in better exception handling on the appliance police fixture as a bonus.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit f70d752a21da5774ccc85ba7bc03512f84750b87:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: conftest.py [hunk 1]
```diff
@@ -79,15 +86,14 @@ def appliance_police():
...
                                    timeout=60).status_code
         if status_code != 200:
-            raise Exception('Status code was {}, should be 200'.format(status_code), port)
-    except Exception as e:
-
-        port = e.args[1] if len(e.args) == 2 else None
-        if port == 443:
+            raise _AppliancePoliceException('Status code was {}, should be 200'.format(
+                status_code), port)
+    except _AppliancePoliceException as e:
```
Comment (psav) [inline conftest.py]: This is great, except, it still doesn't handle the possibility that we get a different error during the ```try:```

Comment (seandst) [inline conftest.py]: That's intentional.

Comment (seandst) [inline conftest.py]: Do you want this to be a diaper?

Comment (psav) [inline conftest.py]: That's what started a pacman slave last time....

**Comment by dajoRH** [comment]
Lint report for commit b04a786ada344dd811a0f87e351873ec2813ce72:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by psav** [comment]
PRT is happy :cake: :facepunch:

**Pull request merged**